Codeinwp/otter-blocks
# Add Font preview

Closes #1488 .


### Summary


Add Font preview for Font Family Select component.

### Screenshots 

https://user-images.githubusercontent.com/17597852/219354379-ee8ec68b-7063-4d8b-8807-7074d5c4ec6b.mp4

----

### Test instructions


ℹ️ Since it uses the Google API for optimized fonts, some browser settings and extensions block the request. If the browser which you are testing is aggressively blocking Google API, you will see in the console errors like `ERR_BLOCKED_BY_CLIENT` 

ℹ️ Making the request takes time; each font loading has 2 requests. It might be faster or slower, depending on your internet speed and CPU. Check the video to see the effect.

ℹ️ First time, it will look slow, but after, the browser will cache the request so that the next time it will be much faster. This happens only if you don't disable the cache.

ℹ️ Not all fonts will be available; depending on the server setup, you will see errors about CORS policy.

ℹ️ Make sure to publish the page if you want to see the font in Frontend.



---- 

### Checklist before the final review
